src/midje/Bootstrap.clj

```diff
@@ -1,23 +0,0 @@
-(ns midje.Bootstrap)
-
-(defonce bootstrapped false)
-
-(defn bootstrap []
-  (when-not bootstrapped
-    (require 'midje.config)
-    (let [saved-ns (ns-name *ns*)]
-      (try
-        (in-ns 'midje.config)
-        ((ns-resolve 'midje.config 'load-config-files))
-      (finally
-        (in-ns saved-ns))))
-
-    (require 'midje.emission.api)
-    ( (ns-resolve 'midje.emission.api 'load-plugin)
-      ( (ns-resolve 'midje.config 'choice) :emitter))
```

Comment:
@marick if you have a moment, would be very helpful to hear the context behind the usage of `ns-resolve` here. Is there a subtly I'm missing, or is it used to deal with constraints that have since shifted? Thanks!